pyvista/plotting/renderer.py

```diff
@@ -917,9 +917,9 @@ def add_axes(
         labels_off : bool, optional
             Enable or disable the text labels for the axes.
 
-        marker_args : dict, optional
-            Parameters for the orientation marker widget. See the parameters of
-            :func:`pyvista.create_axes_marker`.
+        marker_args : dict, optional 
+            DEPRECATED: Please use **kwargs for passing parameters for 
```

Comment:
We don't seem to be consistent about this, so optionally consider adding a deprecated directive:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated